lms/templates/courseware/legacy_instructor_dashboard.html

```diff
@@ -10,7 +10,6 @@
 %>
 
 <%block name="pagetitle">${_("Legacy Instructor Dashboard")}</%block>
-<%block name="nav_skip">#instructor-dashboard-content</%block>
```

Comment:
There's an [`a.nav-skip`](https://github.com/edx/edx-platform/blob/6d2468d0dcf782a7561ecc61946146d6fae2762f/lms/templates/main_django.html#L31) at the very top of the `main_django.html` template.